src/watchFiles.ts

```diff
@@ -1,14 +1,16 @@
-require('dotenv').config();
-const fs = require('fs');
-const path = require('path');
-const chokidar = require('chokidar');
-const chalk = require('chalk');
+import dotenv from 'dotenv';
+import fs from 'fs-extra';
```

Comment:
You shouldn't need this package (assuming you added it for async support), node has it's own async versions of fs methods. https://nodejs.org/api/fs.html#promises-api

Reply:
`fs-extra` is a drop in replacement for `fs` and/or `fs/promises`, it provides a `pathExists` function which is using `fs.access` along with some other checks, it wraps it all up into one call

Reply:
[I wouldn't use it, `pathExists` is literally just `access` wrapped in a promise](https://github.com/jprichardson/node-fs-extra/blob/master/lib/path-exists/index.js), not worth adding a dependency to do that, write it yourself. You should have a very good reason to add any dependency, this doesn't seem one.